boards/arm/nrf5340dk_nrf5340/doc/index.rst

```diff
@@ -200,6 +209,33 @@ The process requires the following steps:
 4. Build the application firmware for the Network MCU using
    ``-DBOARD=nrf5340pdk_nrf5340_cpunet``.
 
+
+Building the Secure firmware with TF-M
+--------------------------------------
+
+The process to build the Secure firmware image using TF-M and the Non-Secure
+firmware image using Zephyr requires the following steps:
+
+1. Build the Non-Secure Zephyr application
+   for the Application MCU using ``-DBOARD=nrf5340pdk_nrf5340_cpuappns`` and
+   ``CONFIG_BUILD_WITH_TFM=y`` in the application project configuration file.
+   The Zephyr build system will perform the following steps automatically:
+
+      * Build the Non-Secure firmware image as a regular Zephyr application
+      * Build a TF-M (secure) firmware image
+      * Optionally build a bootloader image (MCUboot)
+      * Merge the output binaries together
```

Comment:
I'm not sure if it will be clear that only the two firmware images are merged together and the bootloader one is not. Maybe it would be better to change the order of the last two points?

Reply:
Yes! that's good point 👍

Reply:
done, and also for nrf9160